apps/site/.cloudflare/node/fs/promises.mjs

```diff
@@ -0,0 +1,28 @@
+const cloudflareContextSymbol = Symbol.for('__cloudflare-context__');
+
+export async function readFile(path) {
+  console.log('fs/promies#readFile', path);
+
+  const { env } = global[cloudflareContextSymbol];
+
+  const text = await env.ASSETS.fetch(
```

Comment:
I recall the link on the right can be anything, but could you add a documentation?

Also please don't mix await/Promises.

Reply:
> I recall the link on the right can be anything, but could you add a documentation?

Yes we only care about the url's path, I'll clarify that 👍 

> Also please don't mix await/Promises.

Good call 👍